infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/VerticalGeometryListing.kt

```diff
@@ -202,10 +223,10 @@ private val commonVerticalGeometryListingCsvEntries = arrayOf(
         }
     },
     CsvEntry(translateVerticalGeometryListingHeader(VerticalGeometryListingHeader.HEIGHT_START)) {
-        formatTo3Decimals(it.start.height)
+        it.start.height
```

Comment:
Näitä en tosiaan lähtenyt rivittämään käsin. IDEA rivittäisi ne taas tällä tavalla uudestaan heti kun joku sitä nämä seuraavaksi pyytäisi rivittämään.